apps/editor/src/app/routes/blockStyles/blockStyleList.tsx

```diff
@@ -0,0 +1,379 @@
+import {ApolloError} from '@apollo/client'
+import styled from '@emotion/styled'
+import {
+  BlockStyle,
+  BlockType,
+  getApiClientV2,
+  useBlockStylesQuery,
+  useCreateBlockStyleMutation,
+  useDeleteBlockStyleMutation,
+  useUpdateBlockStyleMutation
+} from '@wepublish/editor/api-v2'
+import {
+  createCheckedPermissionComponent,
+  IconButtonTooltip,
+  ListViewActions,
+  ListViewContainer,
+  ListViewHeader,
+  PermissionControl,
+  TableWrapper
+} from '@wepublish/ui/editor'
+import {equals} from 'ramda'
+import {memo, useCallback, useReducer, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {MdAdd, MdDelete, MdSave} from 'react-icons/md'
+import {
+  Button,
+  CheckPicker,
+  FlexboxGrid,
+  Form,
+  IconButton as RIconButton,
+  Loader as RLoader,
+  Message,
+  Modal,
+  toaster
+} from 'rsuite'
+
+const FlexGridSmallerMargin = styled(FlexboxGrid)`
+  margin-bottom: 12px;
+  gap: 12px;
+`
+
+const Content = styled.div`
+  margin-top: 2rem;
+  height: 100%;
+`
+
+const IconButton = styled(RIconButton)`
+  margin-left: 12px;
+`
+
+const Flex = styled.div`
+  flex: 0 0 auto;
+`
+
+const FlexWrapper = styled.div`
+  max-width: 300px;
+  flex: 1 1;
+`
+
+const Loader = styled(RLoader)`
+  margin: 30px;
+`
+
+enum BlockStyleListActionType {
+  Set = 'set',
+  Create = 'create',
+  Update = 'update',
+  Delete = 'delete'
+}
+
+type BlockStyleListActions =
+  | {type: BlockStyleListActionType.Set; payload: Record<string, BlockStyle>}
+  | {type: BlockStyleListActionType.Create; payload: BlockStyle}
+  | {
+      type: BlockStyleListActionType.Update
+      payload: BlockStyle
+    }
+  | {type: BlockStyleListActionType.Delete; payload: {id: string}}
+
+const mapBlockStyleToFormValue = (blockStyles: BlockStyle[] | null | undefined) =>
+  blockStyles?.reduce((obj, node) => {
+    obj[node.id] = node
+
+    return obj
+  }, {} as Record<string, BlockStyle>) ?? {}
+
+const blockstyleFormValueReducer = (
```

Comment:
blockStyle <-- capital S

Reply:
Will be done in the next PR! Thanks for finding